### PR TITLE
Portfolio: Toolbar and Headline section

### PR DESCRIPTION
**Pull request by olaishola05**
# First stage of the portfolio project

- Add setup configs for linters
- Add index.html, style.css, and image folder
- Add navigation logo and the hamburger icon for the menu bar
- Add the Headline section with title and social icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
```
Comment (carlos-ssh) [inline index.html:21]: - The container of the section is a ``<header>`` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      <nav class="navbar">
+        <ul>
+          <li class="logo"><a href="#">Olaish</a></li>
+          <li class="menu"><i class="fas fa-bars"></i></li>
+        </ul>
+      </nav>
+      <div class="hero">
+        <section class="headline">
+          <div class="title">
+            <h2>I’m Ola Glad to see you!</h2>
```
Comment (carlos-ssh) [inline index.html]: - The main title (e.g., "I'm Ola ...") is a ``<h1>`` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="social">
+            <p>LET’S CONNECT</p>
```
Comment (carlos-ssh) [inline index.html]: - The button is a link ``<a>`` or ``<button>`` tag, and follow the button styling: Active, Hover and Pressed.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @netman5 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (olaishola05) [reply index.html:21]: Hi Carlos!
Pls I don't quite get the header part, did u mean the wrapper container should be a header tag? Thanks

**Review by rustam-tolipov — APPROVED** [review]
## Hi @netman5  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**